csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3623,7 +3623,7 @@ The relevant path for this test is:
   ]
 ```
 
-> The list of involements contains two items with the same tuple `party`, `status` and `date`.
+> The list of involvements contains two items with the same tuple `party`, `status` and `date`.
```

Comment:
As the state is not important for this I would suggest the following:
```suggestion
> The list of involvements contains two items with the same tuple `party` and `date`.
```

Moreover, I suggest to change the first status into `completed` to prevent misinterpretation of the example.

Reply:
See https://github.com/sthagen/csaf/pull/59.